CyrillicFilters/RussianFilter/sections/specific.txt

```diff
@@ -24,6 +24,7 @@ happycoin.club##a[href][target="_blank"][rel="noopener"] > img[width][height]
 happycoin.club#?#.theiaStickySidebar > div.widget_text:has(> div.widget-title-cover > h5:contains(Реклама))
 novate.ru#?##right > h2:contains(Рекомендуем)
 ||myfanstv.net/uploads/*/baner/
+leingpt.ru#@##reklama
```

Comment:
We add anti-adblock rules here (comment with issue link + rule/s):
https://github.com/AdguardTeam/AdguardFilters/blob/5136e7865d95a5882c81da8aa35795bea7e236f8/CyrillicFilters/RussianFilter/sections/antiadblock.txt#L28